charts/selenium-grid/Chart.yaml

```diff
@@ -2,6 +2,6 @@ apiVersion: v2
 name: selenium-grid
 description: A Helm chart for creating a Selenium Grid Server in Kubernetes
 type: application
-version: 0.15.8
+version: 0.15.9
```

Comment:
I would move this to 0.16.0 since there is a new feature.

Reply:
Makes sense I updated the value to 0.16.0